mersenne-31/src/mersenne_31.rs

```diff
@@ -17,7 +17,7 @@ use rand::Rng;
 use serde::{Deserialize, Serialize};
 
 /// The Mersenne31 prime
-const P: u32 = (1 << 31) - 1;
+pub(crate) const P: u32 = (1 << 31) - 1;
```

Comment:
can't use `Mersenne31::ORDER_U32`? I'm not convinced we should be exposing one constant in multiple ways. Although it's just `pub(crate)`, so I'm not too worried.

Reply:
Yep that's a good point.